project.py

```diff
@@ -462,9 +585,10 @@ def start_processing(self, input_file):
 
 
 def main(input_file, slide_number=0, start_frame=0, check_interval=30,
-         sim=0.70):
+         sim=0.60, compare_func='dhash', dbg=True):
```

Comment:
This ought to be called `debug{
` rather than `dbg` -- you risk that it will not be picked up as a keyword argumentm